cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsPlatformResources.java

```diff
@@ -841,50 +842,49 @@ public CloudEncryptionKeys encryptionKeys(ExtendedCloudCredential cloudCredentia
+                                        Map<String, Object> meta = new HashMap<>();
+                                        meta.put("aWSAccountId", describeKeyResult.getKeyMetadata().getAWSAccountId());
+                                        meta.put("creationDate", describeKeyResult.getKeyMetadata().getCreationDate());
+                                        meta.put("enabled", describeKeyResult.getKeyMetadata().getEnabled());
+                                        meta.put("expirationModel", describeKeyResult.getKeyMetadata().getExpirationModel());
+                                        meta.put("keyManager", describeKeyResult.getKeyMetadata().getKeyManager());
+                                        meta.put("keyState", describeKeyResult.getKeyMetadata().getKeyState());
+                                        meta.put("keyUsage", describeKeyResult.getKeyMetadata().getKeyUsage());
+                                        meta.put("origin", describeKeyResult.getKeyMetadata().getOrigin());
+                                        meta.put("validTo", describeKeyResult.getKeyMetadata().getValidTo());
+
+                                        if (!CloudConstants.AWS.equalsIgnoreCase(describeKeyResult.getKeyMetadata().getKeyManager())) {
+                                            CloudEncryptionKey key = new CloudEncryptionKey(
+                                                    item.getKeyArn(),
+                                                    describeKeyResult.getKeyMetadata().getKeyId(),
+                                                    describeKeyResult.getKeyMetadata().getDescription(),
+                                                    keyListEntry.getAliasName().replace("alias/", ""),
+                                                    meta);
+                                            cloudEncryptionKeys.getCloudEncryptionKeys().add(key);
+                                        }
+                                    });
+                        } catch (AmazonServiceException e) {
+                            if (e.getStatusCode() == UNAUTHORIZED) {
+                                String policyMessage = "Could not get encryption keys because the user does not have enough permission.";
+                                LOGGER.error(policyMessage, e);
+                            } else {
+                                LOGGER.info(queryFailedMessage, e);
+                            }
+                        } catch (Exception e) {
+                            LOGGER.warn(queryFailedMessage, e);
+                        }
                     }
-                } catch (Exception e) {
-                    LOGGER.warn(queryFailedMessage, e);
-                }
-            }
         } catch (AmazonServiceException ase) {
             if (ase.getStatusCode() == UNAUTHORIZED) {
                 String policyMessage = "Could not get encryption keys because the user does not have enough permission.";
```

Comment:
Similarly, I think this message could be extended to say listKeys threw an error